Cargo.toml

```diff
@@ -173,7 +173,6 @@ backoff = { version = "0.4.0", features = [ "tokio" ] }
 base64 = "0.22.0"
 bb8 = "0.8.3"
 bcs = "0.1.6"
-bincode = "1.3.3"
```

Comment:
Weird! I definitely didn't _intend_ to do that. I'll investigate a bit.

Reply:
I'm very confused how this happened. I checked out the file from `main` to be sure this isn't due to some change I made unintentionally. It doesn't appear to be, since after checking the lockfile remains unchanged. I will remove this, using the one from `main`. Thanks for catching this.